dist/navigation-timing.js

```diff
@@ -24,6 +23,8 @@ function postMetrics(metrics) {
 function getMetrics() {
     var now = new Date(),
         metrics = {
+            complete: '',
+            navigator: (window.navigator ? window.navigator : {}),
```

Comment:
Currently passing the navigator object as is. We will probably refine this later.